calitp/auth.py

```diff
@@ -0,0 +1,35 @@
+import os
+from typing import Sequence
+
+import google_crc32c
+from google.cloud import secretmanager
+
+AUTH_KEYS_ENV_VAR = "CALITP_AUTH_KEYS"
```

Comment:
can you explain (either in PR body or somehow else) the intended use case for this? I assume that this could make it so we don't have to make a code change every time we get a new API key (could just add the new key name to the env var).... are we already using this for RT or Airflow?

Reply:
Yes I will change. We aren't using it anywhere currently.